src/Saritasa.Git.GitFlowStatus/GetGitFlowStatusCmdlet.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+using LibGit2Sharp;
+
+namespace Saritasa.Git.GitFlowStatus
+{
+    public enum BranchType
+    {
+        // ReSharper disable once InconsistentNaming
```

Comment:
PowerShell enums has the same PascalCase style as other .NET enums.

    PS C:\Users\anton> [System.Enum]::GetNames([System.Management.Automation.ActionPreference])
    SilentlyContinue
    Stop
    Continue
    Inquire
    Ignore
    Suspend